recipes/radiospectra/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "radiospectra" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "94840b28d38b6a811bec8e601a9bcf9d789e44fbeb3ebf173d303fcb38e7864a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sunpy >=0.8
```

Comment:
Is this a build dep?

Reply:
No, you are correct! I missed that one!!